### PR TITLE
Fix: libwdi-sys build failures

### PR DESCRIPTION
**Pull request by dragonmux**
This PR fixes an issue found in the libwdi-sys crate only after the initial publication, and prepares the repository for re-publication of the crate updated with the fix.

### CODE REVIEW

File: libwdi-sys/build.rs [hunk 1]
```diff
@@ -281,13 +275,13 @@ impl LibwdiBuild
...
         // Again relative from libwdi_repo (after that .map() below, at least).
         let mut as_is: Vec<PathBuf> = [
+            "libwdi.c",
             "libwdi_dlg.c",
             "logging.c",
             "pki.c",
             "tokenizer.c",
             "vid_data.c",
             "embedder.c",
-            "installer.c",
```
Comment (esden) [inline libwdi-sys/build.rs:290]: Was that removed on purpose? Just want to make sure. :)

Comment (dragonmux) [reply libwdi-sys/build.rs:290]: It was - the `as_is` array and the `needs_patch` array above are supposed to be mutually exclusive to each other (we hadn't realised this before) so as it's mentioned in `needs_patch` it must not be mentioned in `as_is`.

Comment (esden) [reply libwdi-sys/build.rs:290]: Ok sounds good.

**Review by esden — APPROVED** [review]
LGTM

**Pull request merged**